### DISCUSSION TITLE
link about the code

### DISCUSSION THREAD

**Discussion by orthodontics3d** [Markram71/Bitwig-Performance-Twister#9] [General]
Hello, I would like a link to the code so that I can make changes to my workflow. best regards

**Comment by Markram71** [comment]
Hi, 
here's the link to the source code. https://github.com/Markram71/Bitwig-Performance-Twister/tree/main/src

Good luck and cheers
MK

**Comment by orthodontics3d** [comment]
Thanks a lotLe 11 oct. 2024 à 07:47, Dr. Martin Kramer ***@***.***> a écrit :﻿
Hi,
here's the link to the source code. https://github.com/Markram71/Bitwig-Performance-Twister/tree/main/src
Good luck and cheers
MK
